libs/service-portal/settings/access-control/src/constants.ts

```diff
@@ -0,0 +1,3 @@
+// To keep this deprecated UI future compatible with graphql schema changes, we're
+// hard-coding the domain it should manage.
+export const ISLAND_DOMAIN = '@island.is'
```

Comment:
You are defining the same constant in two different libraries. Put it in a shared constant library?

Reply:
Good idea. Didn't want to create a new library but I see we already have a shared constant library.